### DISCUSSION TITLE
Two new budibase plugins : radar and bubble charts

### DISCUSSION THREAD

**Discussion by juliencognet** [Budibase/budibase#9469] [Contributing]
Hello, 

I have created two new plugins for budibase : a radar chart and a bubble chart.
They are available on my Github:
- https://github.com/juliencognet/budibase-bubble-chart
- https://github.com/juliencognet/budibase-radar-chart

They perfectly work for me. You are free to add it to the curated list of plugins if you approve them.

I really appreciated the developer experience. Therefore, I am volunteer to go further and contribute to budibase.

Best regards,

Julien COGNET

**Comment by melohagan** [comment]
Hey @juliencognet 

These charts look really cool! There is quite a lot going on there, for example the *Advanced Options*. An example would be sweet. 

Could you also export some tables that you used for testing (as CSVs) just so I can give it a whirl myself? 

Great job though - the screenshots look very nice! 😄 Glad you had a good dev experience as well!

**Reply by juliencognet** [reply]
Thanks for your reply. Il will rework documentation of the plugins this week-end, to give more details on each parameter and samples of charts that can be built.

**Reply by melohagan** [reply]
No rush, but looking forward to it! 😄

**Comment by PlunetBusinessManager** [comment]
I was exactly looking for a bubble chart and you already did it. THANKS SO MUCH!

**Comment by SidPayton** [comment]
Hi @juliencognet 
hope you are doing well. 

Your plugins are what made me choose Budibase to begin with. Thanks for your hard work.
Do you think it would be possible to add an X-axes description and the ability to click on the bubbles?
That would make this plug in essential to me.

**Reply by juliencognet** [reply]
Hi @SidPayton, I am happy that my little contribution helped another team. About your request, it is really easy to add the X-axes description and I will try to implement it as soon as possible, but the ability to click on the bubbles is more tricky and needs refinement on your need. When we mouse over the bubble, a tooltip is displayed. I could add a link inside the tooltip but then two elements must be set: the text of the link and the url itself which should be an expression including an id. Could you please define your requirement ?

**Reply by SidPayton** [reply]
Hi @juliencognet, sorry for the late reply, I didn't set my notifications right. 
First off, thanks for your willingness to help out. I appreciate that very much.

From a UX perspective I think it would be great if we could click the bubble itself. But I'm already happy with your solution, if that's too difficult to implement. 

I attached a screenshot of what I should be able to do within Budibase when I programm this. As I want the Bubbles to be another way of navigating otherwise complex data I just want to click the bubble and read out that particular row of my database and show and display that row in more detail just as I am able to do, when I use the normal table. 

Ideally within the bubbles-setting there would be a an option such as "On bubble click" that  lets me set all different kind of actions just like its possible in tables (see attached screenshot). 

![image](https://github.com/Budibase/budibase/assets/5787625/bf247f7b-d329-4853-8af3-e1c3cc2450d0)

I hope this clarifies it a bit more. Let me know if I can help out any further. I will check my messages more often. ;)

**Reply by juliencognet** [reply]
Hello, unfortunately, due to the change of licence strategy chosen by budibase team, I will not invest any more in budibase product as it cannot be used by more than 20 users (in open source version). Therefore, I will not develop a better version of the plugin.
